docs/sources/administration/configuration.md

```diff
@@ -179,7 +179,7 @@ Directory where Grafana automatically scans and looks for plugins. Manually or a
 
 ### provisioning
 
-Folder that contains [provisioning]({{< relref "provisioning.md" >}}) config files that grafana will apply on startup. Dashboards will be reloaded when the json files changes
+Folder that contains [provisioning]({{< relref "provisioning.md" >}}) config files that grafana will apply on startup. Dashboards will be reloaded when the json files changes.
```

Comment:
Looks good. I have a request. Can we change grafana to Grafana. Thank you very much!

Reply:
Yes, Pls..Thanks,
Regards,
Ashish

Reply:
Will you be kind and make the change? Thank you again!

Reply:
Hi, I have raised separate pull request (28231) for the same.

Thanks,
Ashish